src/InfluxDB2/QueryApi.php

```diff
@@ -64,13 +77,20 @@ public function query($query, $org = null, $dialect = null)
     }
 
     /**
-     * @param $query
-     * @param $org
-     * @param $dialect
+     * Executes the Flux query against the InfluxDB 2.0 and returns generator to stream the result.
+     *
+     * @param string| Query $query
+     * @param string|null $org
+     * @param Dialect|null $dialect
+     *
      * @return FluxCsvParser generator
      */
-    public function queryStream($query, $org = null, $dialect = null): ?FluxCsvParser
+    public function queryStream($query, string $org = null, Dialect $dialect = null): ?FluxCsvParser
```

Comment:
Please be consistent. Mark all nullable types with `?` or remove `?` from `?FluxCsvParser`.

```suggestion
    public function queryStream($query, ?string $org = null, ?Dialect $dialect = null): ?FluxCsvParser
```